Arrowgene.Ddon.GameServer/Handler/QuestGetLightQuestListHandler.cs

```diff
@@ -15,7 +19,26 @@ public QuestGetLightQuestList(DdonGameServer server) : base(server)
 
         public override void Handle(GameClient client, StructurePacket<C2SQuestGetLightQuestListReq> packet)
         {
-            client.Send(new S2CQuestGetLightQuestListRes());
+            // client.Send(new S2CQuestGetLightQuestListRes());
+
+            S2CQuestGetLightQuestListRes res = new S2CQuestGetLightQuestListRes();
+            foreach (var questId in client.Party.QuestState.GetActiveQuestIds())
+            {
+                var quest = QuestManager.GetQuest(questId);
+                if (quest.QuestType == QuestType.PersonalQuest || quest.QuestType == QuestType.ExtremeMissions)
```

Comment:
This might've been something which I copied from your original PR and just carried it over as I modified the systems. I can remove it if you think it does not belong here.

Reply:
Yeah, adding EM in the quest board was a hack to test if we could do EMs. It doesn't belong there

Reply:
Done